src/components/AuthNav/AuthNav.jsx

```diff
@@ -0,0 +1,15 @@
+import "./AuthNav.scss";
+import Logo from "../Logo/Logo";
+import Button from "../Button/Button";
+
+const AuthNav = ({isLogin}) => {
+    
+    return (
+        <div className="authNav">
+            <Logo isTextDark={true}/>
+            <Button buttonText={isLogin ? "Login" : "Sign Up"} handleClick={""}/>
```

Comment:
Instead of passing an empty string for handleClick coulld you just pass null - handleClick={null}.
This stops the error message when clicking on the button.

Reply:
We were going to replace that handleclick with functional routing components so the button actually works, but we ran out of time. As of present, we're trying to fix the issue without wiping all our progress.